### PR TITLE
DWR-1406 Added caseInsensitiveValue to Subject.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

**Comment by agorina** [comment]
Merging, but will address comments if needed.

File: model/src/test/java/org/opentestsystem/rdw/common/model/SubjectTest.java [hunk 1]
```diff
@@ -29,11 +35,6 @@ public void itShouldConvertFromString() {
         assertThat(Subject.valueOf("ELA")).isEqualTo(Subject.ELA);
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void stringValueIsCaseSensitive() {
-        Subject.valueOf("ela");
-    }
-
```
Comment (malaffoon) [inline model/src/test/java/org/opentestsystem/rdw/common/model/SubjectTest.java]: why did you remove this test? Doesn't it demonstrate that valueOf() is case sensitive?

Comment (agorina) [reply model/src/test/java/org/opentestsystem/rdw/common/model/SubjectTest.java]: Oops, I do not know :)  I will restore it back.
